### ISSUE TITLE
Add `.get_measured_scatter()` function to determine the wavelength-by-wavelength scatter in a Rainbow.

### ISSUE THREAD

**Issue by zkbt** [zkbt/chromatic#44]
It'd be great to have a function that calculates a wavelength-like array containing an estimate of the measured scatter across time points, for each wavelength. 

It might look something like `def get_measured_scatter(rainbow, method='standard-deviation'):`, where the user is allowed to pass in an option to describe how the scatter should be estimated. Possible options might be:
- simple standard deviation. This means a lot statistically, but it will be really sensitive to outliers.
- 1.48*MAD. The median absolute deviation from the median (MAD) is a nice outlier-resistant estimate of the scatter. Multiplying it by 1.48 makes it numerically closer to the standard deviation under ideal assumptions (so can be sort of compared directly)
- some clever outlier-resistent scatter estimate (e.g. standard deviation with outlier rejection)

To start developing this, the simplest approach is probably to just write a function in a notebook. Test it on some Rainbow object, and then once it behaves how you like, we can drop it into the code for the `chromatic` package. (I can post more specific instructions on how to do that below.) 

So, to get started, you might open a notebook and run  

```
from chromatic import *
rainbow = SimulatedRainbow().inject_transit()

def ...
```

and then start drafting your function.

**Comment by zkbt** [comment]
@kortizceballos and @zkbt are working on this one!

**Comment by zkbt** [comment]
Closed by #112 . Hooray!!!
